auth/ed25519.go

```diff
@@ -0,0 +1,28 @@
+package auth
```

Comment:
we need // +build evm
at top of the file, and then have a stub one
anything that requires the EVM has to have a build tag